public/css/main.css

```diff
@@ -30,27 +31,28 @@
     bottom: 1px;
     top: auto;
 }
-@media screen and (min-width:1333px) {
+@media screen and (min-width: 1280px) {
     #forkongithub {
         position: absolute;
         display: block;
+        overflow: hidden;
         top: 0;
         left: 0;
         width: 200px;
-        overflow: hidden;
         height: 200px;
-        z-index: 0;
+        z-index: 99;
+        pointer-events: none;
+        transform: rotate(-45deg);
+        -webkit-transform: rotate(-45deg);
```

Comment:
Out of curosity, what is our browser support policy here?

Reply:
Absolutely no idea at all, I simply followed the existing support through the transform prefixes below ([L46](https://github.com/cdnjs/new-website/pull/266/files#diff-4fab20fc9b7238f5f5307386f2b163ffL46)-L53 of old)

Reply:
No problem, might raise an issue for it then later.